py3status/module.py

```diff
@@ -106,6 +106,8 @@ def sleep(self):
     def wake(self):
         self.sleeping = False
         cache_time = self.cache_time
```

Comment:
stupid question, but why assign this to a new variable when we could use self.cache_time wherever needed in this function ?